src/regions/ca/info/ca-pe/reporting.fr.md

```diff
@@ -1,21 +1,21 @@
+A partir du 13 mars 2020, le Bureau principal de la santé publique donne ses conseils : Cesser tout voyage non essentiel à l'extérieur du Canada et s'isoler pendant 14 jours après tout voyage international, y compris aux États-Unis.
 
-À ce moment-ci, tous les voyageurs revenant de zones touchées(link is external) (autres que la province du Hubei ou l’Iran) sont priés de se surveiller de près et de surveiller leurs enfants pendant 14 jours après leur retour chez eux et de composer le 8-1-1 s’ils développent des symptômes (fièvre, toux ou difficultés respiratoires).
+### Quand dois-je appeler le 811 ?
 
-### Que dois-je faire si je suis revenu de la province du Hubei, en Chine, ou de l’Iran au cours des 14 derniers jours?
+Si vous venez de rentrer de l'étranger et que vous présentez des symptômes tels qu'une toux, de la fièvre ou des difficultés respiratoires, appelez le 811 pour subir un dépistage et, si nécessaire, être dirigé vers une clinique pour y subir un test.
 
-Les provinces et territoires recommandent aux voyageurs qui arrivent de s'isoler à leur retour. Ces voyageurs doivent donc rester à la maison et ne pas se rendre à l’école ou au travail. Les voyageurs qui reviennent du Hubei sont encouragés à composer le 8-1-1 pour prendre contact avec les Services infirmiers de santé publique locaux afin d’obtenir des conseils et un soutien supplémentaires. Les Services infirmiers de santé publique peuvent aider les personnes qui ont voyagé à s’assurer qu’elles connaissent les symptômes à surveiller.
+Le nombre d'appels au 811 est plus élevé que la normale. Il faudra plus de temps que d'habitude pour répondre à votre appel. Ils s'efforcent de répondre à tous les appels le plus rapidement possible. Nous vous remercions de votre patience.
 
-### Que dois-je faire si je n’ai pas voyagé récemment, mais que je suis tout de même préoccupé par le COVID-19?
+Si vous répondez OUI à l'une des questions ci-dessous, composez le 811
 
-Si vous n’avez pas voyagé récemment, mais que vous êtes tout de même préoccupé par le coronavirus, veuillez lire les questions suivantes attentivement :
+Au cours des 14 derniers jours, vous avez :
 
-1. Avez-vous eu un contact étroit avec un cas confirmé de nouveau coronavirus au cours des 14 derniers jours?
-2. Au cours des 14 derniers jours, avez-vous eu un contact étroit avec une personne qui fait de la fièvre ou qui souffre d’une maladie respiratoire aiguë (apparition ou exacerbation de la toux ou difficultés respiratoires) qui s’est rendue dans une zone touchée dans les 14 jours avant de tomber malade?
+- voyagé à l'étranger et avez une fièvre supérieure à 38°C ET/OU une toux
+- été en contact étroit (à moins de 2 mètres) avec une personne dont le laboratoire a confirmé le cas de COVID-19 ?
+- Les habitants de l'île qui ont voyagé hors du Canada sont priés de s'isoler pendant les 14 jours suivant leur retour au Canada. Si vous avez de la fièvre ou si vous toussez, appelez le 811.
```

Comment:
Shouldn't this be a separate line instead of a bullet point?

Reply:
fixing 🎸

Reply:
removed a few things since it's redundant